src/modding/ModManager.cs

```diff
@@ -857,6 +1308,251 @@ private void OpenInfoUrlPressed()
+            selectedModPreviewImagesContainer.GetTabCount();
+        galleryRightButton.Disabled = selectedModPreviewImagesContainer.CurrentTab >=
+            selectedModPreviewImagesContainer.GetTabCount() - 1;
+        galleryLeftButton.Disabled = selectedModPreviewImagesContainer.CurrentTab <= 0;
+    }
+
+    private void GalleryLeftArrowPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        selectedModPreviewImagesContainer.CurrentTab -= 1;
+        galleryLabel.Text = (selectedModPreviewImagesContainer.CurrentTab + 1) + "/" +
+            selectedModPreviewImagesContainer.GetTabCount();
+        galleryRightButton.Disabled = selectedModPreviewImagesContainer.CurrentTab >=
+            selectedModPreviewImagesContainer.GetTabCount() - 1;
+        galleryLeftButton.Disabled = selectedModPreviewImagesContainer.CurrentTab <= 0;
+    }
+
+    private void MoveButtonPressed(bool moveUp, int amount)
+    {
+        if (notEnabledMods is null || enabledMods is null)
+        {
+            return;
+        }
+
+        ItemList chosenList;
+        List<FullModDetails> chosenModList;
+
+        if (availableModsContainer.IsAnythingSelected())
+        {
+            chosenList = availableModsContainer;
+            chosenModList = notEnabledMods;
+        }
+        else if (enabledModsContainer.IsAnythingSelected())
+        {
+            chosenList = enabledModsContainer;
+            chosenModList = enabledMods;
+        }
+        else
+        {
+            return;
+        }
+
+        MoveItem(chosenList, chosenModList, moveUp, chosenList.GetSelectedItems()[0], amount);
+
+        var currentIndex = chosenList.GetSelectedItems()[0];
+
+        moveModUpButton.Disabled = currentIndex == 0;
+        moveModDownButton.Disabled = currentIndex >= chosenList.GetItemCount() - amount;
+
+        UpdateOverallModButtons();
+    }
+
+    /// <summary>
+    ///   Handles the movement of the ItemList by any amount
+    /// </summary>
+    private void MoveItem(ItemList list, List<FullModDetails> modList, bool moveUp, int currentIndex, int amount)
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        int newIndex;
+        if (moveUp)
+        {
+            newIndex = currentIndex - amount;
+            if (currentIndex == 0 || newIndex < 0)
+            {
+                return;
+            }
+
+            list.MoveItem(currentIndex, newIndex);
+        }
+        else
+        {
+            newIndex = currentIndex + amount;
+            if (currentIndex == list.GetItemCount() - amount)
+            {
+                return;
+            }
+
+            list.MoveItem(currentIndex, newIndex);
+        }
+
+        var movedMod = modList[currentIndex];
+        modList.RemoveAt(currentIndex);
+        modList.Insert(newIndex, movedMod);
+        UpdateLoadPosition(moveUp ? newIndex : currentIndex);
+    }
+
+    private void ResetPressed()
```

Comment:
Is this a button? I think it is unnecessary clutter to have this in addition to the disable all button. The user can just disable all and apply changes or redo their mod list and then apply.

Reply:
Maybe, it just seems like a common enough operation that I just added a simple button for it. But I can remove it if you think it's too superfluous.

Maybe I should change it to just reset any changes the player had made?

Reply:
Yeah, reset changes is a pretty common button to have. Our options menu has that kind of button.